OIIO/WriteOIIO.cpp

```diff
@@ -365,6 +371,7 @@ class WriteOIIOPlugin
     ChoiceParam* _bitDepth;
     IntParam* _quality;
     DoubleParam* _dwaCompressionLevel;
+    IntParam* _zipCompressionLevel;
```

Comment:
- `zipquality` TIFF compression level
- OpenEXR 3.1.3+ `zip` and `zips` EXR compression level